docs/analytics/export.md

```diff
@@ -71,9 +71,50 @@ This option allows you to customize your export configurations in [Azure](https:
-
-The data will take up to 5 minutes to be shown in Azure Blob Storage.
+Azure Blob storage is a service for storing large amounts of unstructured object data, such as text or binary data, that can be accessed from anywhere in the world via HTTP or HTTPS. You can use Blob storage to expose data publicly to the world, or to store application data privately. Exporting the data to Blob Storage is a good choice for the case where you want to have an extended retention but not necessarily plan to access to the data very often. The data will be exported every minute and a new subfolder will be created. The data will be stored the *year/month/day/hour/minute* format (for example, *https://<blob-storage-account>.blob.core.windows.net/archive/2017/12/09/04/03/logs.v1.data*). The data will take up to 5 minutes to be shown in Azure Blob Storage.
+
+The contents of the blob file is a JSON array of client device logs, that looks like this:
+```
+[
+    {
+        "AppId": "046d56b8-ea26-4653-97ba-12b8f99c3ef5",
+        "Timestamp": "2017-12-09T04:02:53.618Z",
+        "InstallId": "e589a371-ea0c-4479-9a7b-9f834adec040",
+        "MessageType": "EventLog",
+        "IngressTimestamp": "2017-12-09T04:02:57.987Z",
+        "MessageId": "980e21a0-0cbb-48ac-8820-28acf4beb00d",
+        "EventId": "ad980536-e743-48a9-ab7e-cb043602d2c9",
+        "EventName": "log_out",
+        "CorrelationId": "83a2daa9-e5b4-4082-ba4a-ce34b95ab859",
+        "IsTestMessage": "False",
+        "SdkVersion": "1.0",
+        "Model": "PC",
+        "OemName": "Samsung",
+        "OsName": "Android",
+        "OsVersion": "8.1.0",
+        "OsApiLevel": "2",
+        "Locale": "EN",
+        "TimeZoneOffset": "PT2M",
+        "ScreenSize": "320x240",
+        "AppVersion": "1.1.0",
+        "AppBuild": "1",
+        "AppNamespace": "com.microsoft.test",
+        "CarrierName": "AT&T",
+        "CarrierCountry": "US",
+        "CountryCode": "US",
+        "WrapperSdkVersion": "1.0",
+        "WrapperSdkName": "mobilecenter.xamarin",
+        "Properties": "{\"extra_00\":\"5bcacf3598ca44ebbbc99e4488cfc854\",\"extra_01\":\"2673e48867c74d51af8dc24c762a8b28\",\"extra_02\":\"5b76c801e5074cd3a13ea37253b94484\",\"extra_03\":\"c1e76aa252c947d4b4bcd4d1d96a7be6\",\"extra_04\":\"caea50034c4f441a963700fa3cf70d03\"}",
+        "SessionId": "10df497a-4261-4995-b466-3fd77ac47395",
+        "SdkName": "mobilecenter.android",
+        "OsBuild": "2",
+        "WrapperRuntimeVersion": "None",
+        "LiveUpdateDeploymentKey": "stage",
+        "LiveUpdatePackageHash": "dsadsdasd3211321233",
+        "LiveUpdateReleaseLabel": "2.0"
+    }
+]
+```
```

Comment:
JSON schema might be more useful here, or in addition. Here is some to get you started:

```{
  "$id": "http://appcenter.microsoft.com/example.json",
  "type": "array",
  "definitions": {},
  "$schema": "http://json-schema.org/draft-06/schema#",
  "items": {
    "$id": "http://appcenter.microsoft.com/example.json/items",
    "type": "object",
    "properties": {
      "AppId": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/AppId",
        "type": "string",
        "title": "The Appid Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "046d56b8-ea26-4653-97ba-12b8f99c3ef5"
        ]
      },
      "Timestamp": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/Timestamp",
        "type": "string",
        "title": "The Timestamp Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "2017-12-09T04:02:53.618Z"
        ]
      },
      "InstallId": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/InstallId",
        "type": "string",
        "title": "The Installid Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "e589a371-ea0c-4479-9a7b-9f834adec040"
        ]
      },
      "MessageType": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/MessageType",
        "type": "string",
        "title": "The Messagetype Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "EventLog"
        ]
      },
      "IngressTimestamp": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/IngressTimestamp",
        "type": "string",
        "title": "The Ingresstimestamp Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "2017-12-09T04:02:57.987Z"
        ]
      },
      "MessageId": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/MessageId",
        "type": "string",
        "title": "The Messageid Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "980e21a0-0cbb-48ac-8820-28acf4beb00d"
        ]
      },
      "EventId": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/EventId",
        "type": "string",
        "title": "The Eventid Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "ad980536-e743-48a9-ab7e-cb043602d2c9"
        ]
      },
      "EventName": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/EventName",
        "type": "string",
        "title": "The Eventname Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "log_out"
        ]
      },
      "CorrelationId": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/CorrelationId",
        "type": "string",
        "title": "The Correlationid Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "83a2daa9-e5b4-4082-ba4a-ce34b95ab859"
        ]
      },
      "IsTestMessage": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/IsTestMessage",
        "type": "string",
        "title": "The Istestmessage Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "False"
        ]
      },
      "SdkVersion": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/SdkVersion",
        "type": "string",
        "title": "The Sdkversion Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "1.0"
        ]
      },
      "Model": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/Model",
        "type": "string",
        "title": "The Model Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "PC"
        ]
      },
      "OemName": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/OemName",
        "type": "string",
        "title": "The Oemname Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "Samsung"
        ]
      },
      "OsName": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/OsName",
        "type": "string",
        "title": "The Osname Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "Android"
        ]
      },
      "OsVersion": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/OsVersion",
        "type": "string",
        "title": "The Osversion Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "8.1.0"
        ]
      },
      "OsApiLevel": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/OsApiLevel",
        "type": "string",
        "title": "The Osapilevel Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "2"
        ]
      },
      "Locale": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/Locale",
        "type": "string",
        "title": "The Locale Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "EN"
        ]
      },
      "TimeZoneOffset": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/TimeZoneOffset",
        "type": "string",
        "title": "The Timezoneoffset Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "PT2M"
        ]
      },
      "ScreenSize": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/ScreenSize",
        "type": "string",
        "title": "The Screensize Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "320x240"
        ]
      },
      "AppVersion": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/AppVersion",
        "type": "string",
        "title": "The Appversion Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "1.1.0"
        ]
      },
      "AppBuild": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/AppBuild",
        "type": "string",
        "title": "The Appbuild Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "1"
        ]
      },
      "AppNamespace": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/AppNamespace",
        "type": "string",
        "title": "The Appnamespace Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "com.microsoft.test"
        ]
      },
      "CarrierName": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/CarrierName",
        "type": "string",
        "title": "The Carriername Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "AT&T"
        ]
      },
      "CarrierCountry": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/CarrierCountry",
        "type": "string",
        "title": "The Carriercountry Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "US"
        ]
      },
      "CountryCode": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/CountryCode",
        "type": "string",
        "title": "The Countrycode Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "US"
        ]
      },
      "WrapperSdkVersion": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/WrapperSdkVersion",
        "type": "string",
        "title": "The Wrappersdkversion Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "1.0"
        ]
      },
      "WrapperSdkName": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/WrapperSdkName",
        "type": "string",
        "title": "The Wrappersdkname Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "mobilecenter.xamarin"
        ]
      },
      "Properties": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/Properties",
        "type": "string",
        "title": "The Properties Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "{\"extra_00\":\"5bcacf3598ca44ebbbc99e4488cfc854\",\"extra_01\":\"2673e48867c74d51af8dc24c762a8b28\",\"extra_02\":\"5b76c801e5074cd3a13ea37253b94484\",\"extra_03\":\"c1e76aa252c947d4b4bcd4d1d96a7be6\",\"extra_04\":\"caea50034c4f441a963700fa3cf70d03\"}"
        ]
      },
      "SessionId": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/SessionId",
        "type": "string",
        "title": "The Sessionid Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "10df497a-4261-4995-b466-3fd77ac47395"
        ]
      },
      "SdkName": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/SdkName",
        "type": "string",
        "title": "The Sdkname Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "mobilecenter.android"
        ]
      },
      "OsBuild": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/OsBuild",
        "type": "string",
        "title": "The Osbuild Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "2"
        ]
      },
      "WrapperRuntimeVersion": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/WrapperRuntimeVersion",
        "type": "string",
        "title": "The Wrapperruntimeversion Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "None"
        ]
      },
      "LiveUpdateDeploymentKey": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/LiveUpdateDeploymentKey",
        "type": "string",
        "title": "The Liveupdatedeploymentkey Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "stage"
        ]
      },
      "LiveUpdatePackageHash": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/LiveUpdatePackageHash",
        "type": "string",
        "title": "The Liveupdatepackagehash Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "dsadsdasd3211321233"
        ]
      },
      "LiveUpdateReleaseLabel": {
        "$id": "http://appcenter.microsoft.com/example.json/items/properties/LiveUpdateReleaseLabel",
        "type": "string",
        "title": "The Liveupdatereleaselabel Schema",
        "description": "An explanation about the purpose of this instance.",
        "default": "",
        "examples": [
          "2.0"
        ]
      }
    }
  }
}```

[JSON Schema Toolset](https://jsonschema.net/) is useful for editing this.

Reply:
I really don't want to turn this into a 7-screen document. If the schema is important, let's publish it somewhere else and link to it.

Reply:
Agree, I don't think this is needed based on customer questions.